### PR TITLE
Update zh.js & zhtr.js (v5.4.4)

### PR DESCRIPTION
**Pull request by 0v0katai**
Updated to v5.4.4 + consistency changes + fixes

**Workload breakdown**
- zh.js
Self-work

- zhtr.js
50% 0v0katai
50% 1der4

### CODE REVIEW

File: zh.js [hunk 1]
```diff
@@ -547,15 +547,15 @@ windows.controls.taunt2 嘲讽（放屁声）
...
+windows.controls.prop 选择物品（长按）
+windows.controls.propRand 选择任意物品
+windows.controls.propRot 物品旋转切换键
+windows.controls.propRotR 重置物品旋转
 windows.controls.premium0 飞天穿墙
 windows.controls.premium1 无敌
 windows.controls.premium2 自杀
 windows.controls.premium3 瞬间击杀
-windows.controls.reset 重置进度并回到起点（仅限跑酷和速通模式）
+windows.controls.reset 重置进度并回到起点（跑酷等）
```
Comment (0v0katai) [inline zh.js:558]: Here I decided to follow en.js's terms (Parkour etc) instead of adding own investigations

**Comment by 0v0katai** [comment]
> 923 event.doubleXP 此局雙倍職業經驗已開啟

I would keep my own translation as en.js used "for this match" (為此場比賽), although I will remove 獎勵 as it is a bit redundant.

> 927 ranked.switch 回合開始後無法換槍

This one is clearer, yeah.

> 928 ranked.spectate 排位賽觀戰已禁止

Oh yeah, I forgot I used 觀戰 above and not 旁觀. Thanks for reminding me of this, but this sentence sounds a bit strange. Maybe 排位賽中禁止觀戰 is better.

>931 ranked.team.spawn 請先加入隊伍在進入比賽
932 ranked.class.spawn 請先選擇職業在進入比賽

I think you mean "再" and not "在" since the latter word is a common typo.

> 939 modes.prophunt.rotation.on ［物品獵殺］已啟用物品迴轉
940 modes.prophunt.rotation.off ［物品獵殺］已停用物品迴轉
941 modes.prophunt.rotation.reset ［物品獵殺］已重設物品迴轉。

https://hinative.com/zh-TW/questions/3202099

'迴轉' indicates an action which is keep going around start and end at the same point. Here rotation means the camera angle in the Prop's perspective, hence 旋轉.

> 944 commands.host {0} 已是目前服主。

This command (/host) just checks (not changes) the current host, hence "{0} 是目前服主。".

>945 commands.noclip.on 飛天穿牆飛行 - 已啟用
946 commands.noclip.off 飛天穿牆飛行 - 已停用

Double meaning for 飛天 and 飛行. I would drop 穿牆 and just use "飛天模式" though.

**Comment by mty22** [comment]
ping when ready to merge

**Comment by mty22** [comment]
@0v0katai @YureiOwO we good to merge?

**Comment by 0v0katai** [comment]
@mty22 Good to merge ^^
image

**Comment by mty22** [comment]
stellar work as usual fellas.

**Pull request merged**